src/main/java/seedu/address/logic/commands/DeleteCategoryCommand.java

```diff
@@ -0,0 +1,76 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_CATEGORY;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Entry;
+import seedu.address.model.person.Person;
+
+/**
+ * Deletes one existing category of an existing person in the address book.
+ */
+public class DeleteCategoryCommand extends Command {
+
+    public static final String COMMAND_WORD = "deleteCategory";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Deletes the category of person identified by the index number used in the displayed person list.\n"
+            + "Parameters: INDEX (must be a positive integer) "
+            + PREFIX_CATEGORY + "CATEGORY\n"
+            + "Example: " + COMMAND_WORD + " 1 "
+            + PREFIX_CATEGORY + "Email";
+
+    public static final String MESSAGE_DELETE_CATEGORY_SUCCESS = "Deleted Category %1$s of Person %2$s";
+    public static final String MESSAGE_CATEGORY_DOES_NOT_EXIST = "Category doesn't exist.";
+
+    public static final String MESSAGE_WRONG_CATEGORY = "This person does not have this category.";
+
+    private final Index targetIndex;
+
+    private final String category;
+
+    /**
+     * @param targetIndex of the person in the filtered person list to edit
+     * @param category    the category needed to be deleted
+     */
+    public DeleteCategoryCommand(Index targetIndex, String category) {
+        this.targetIndex = targetIndex;
+        this.category = category;
+    }
+
+    @Override
```

Comment:
Good method of abstraction, however, please add some description as to what it does.